app/src/main/java/edu/uncc/hw04/fragments/ViewDrinksFragment.java

```diff
@@ -38,12 +44,19 @@ public void onCreate(Bundle savedInstanceState) {
 
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.fragment_viewdrinks, container, false);
+        return inflater.inflate(R.layout.fragment_viewdrinks_list_row, container, false);
```

Comment:
I'd have to look, but I'm fairly certain this should be reverted. Isn't the list row xml supposed to be set on the `ViewDrinksRecyclerAdapter` below?